sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/SymmetricHashJoinStateManager.scala

```diff
@@ -447,6 +447,9 @@ class SymmetricHashJoinStateManager(
       keyToNumValuesMetrics.memoryUsedBytes + keyWithIndexToValueMetrics.memoryUsedBytes,
       keyWithIndexToValueMetrics.customMetrics.map {
         case (metric, value) => (metric.withNewDesc(desc = newDesc(metric.desc)), value)
+      } ++ keyToNumValuesMetrics.customMetrics.collect {
+        case (metric: StateStoreInstanceMetric, value) =>
```

Comment:
nit: add comment e.g. `For instance metric, we want to collect metrics for both state stores. Unlike other custom metrics where it is only collected for keyWithIndexToValue`